### ISSUE TITLE
Priors documentation

### ISSUE THREAD

**Issue by strengejacke** [glmmTMB/glmmTMB#1006]
Currently, the docs for the `coef` argument in `priors` says:

>  The exception is for fixed-effect parameter vectors, where the intercept (if present) is not included; the prior on the intercept must be set explicitly.

We discussed whether or not the intercept should be excluded by default for other components as well - not sure how you decided, but if so, the documentation is not up to date, right?

**Comment by strengejacke** [comment]
Yes, makes sense. It's probably my lack of statistical education that I sometimes struggle with common statistical terms - but also to avoid possible confusion with econometrics and their fixed effects, this clarification might be helpful.

**Comment by bbolker** [comment]
Now reads

> If the ‘class’ element is missing, or a particular element is blank, then all of the elements of the specified parameter vector use independent priors with the given specification. The exception is for the fixed-effect parameter vectors ("fixef", "fixef_zi", "fixef_disp"), where the intercept (if present) is not included; the prior on the intercept must be set explicitly.


And there's an example:

```
     ## no prior is set for the intercept in this case - see Details above
     prior3 <- data.frame(prior = "t(0, 3, 3)",
                          class = "fixef")
     update(g1, priors = prior3)
```

Does that address your confusion?

**Comment by strengejacke** [comment]
Yes, that's clearer! Thanks!
